### PR TITLE
Placing git-lfs documentation in right place

### PR DESCRIPTION
**Pull request by jasonnicholson**
The PKGBUILD file was updated to move the html documentation to the right location.

@dscho here is your pull request per our conversation git-for-windows/git#4327.

### CODE REVIEW

**Review by dscho — COMMENTED** [review]
Thank you for working on this.

Before accepting the PR, I would like to ask for two things:

- Please use your real email address in the Signed-off-by line. The idea is that people can contact you in case of, say, proposed license changes. A `.noreply.github.com` address won't cut it.
- Please increment the `pkgrel` number, so that we can deploy a new package version in this Pull Request.

File: mingw-w64-git-lfs/PKGBUILD [hunk 1]
```diff
@@ -48,4 +48,8 @@ package() {
   install -m755 $srcdir/$folder/git-lfs$exesuffix.exe $pkgdir/$MINGW_PREFIX/bin/git-lfs.exe
   install -d -m755 $pkgdir/$MINGW_PREFIX/share/doc/git-lfs
   install -m755 $srcdir/$folder/README.md $pkgdir/$MINGW_PREFIX/share/doc/git-lfs/README.md
+  for f in $srcdir/$folder/man/html/git-lfs*.1.html; do
+    ff=${f##*/}
+    install -m755 $f $pkgdir/$MINGW_PREFIX/share/doc/git-core/${ff%.1.html}.html
```
[the remark this answers is not in the document]
Comment (dscho) [reply mingw-w64-git-lfs/PKGBUILD:55]: Oh, I see, there are now two commits, one that introduces flawed code, the other one fixing it and bumping pkgrel.

Could we please instead have two commits where one patches the `package()` function (squashing in the fix), and the second commit that bumps `pkgrel`?

[the remark this answers is not in the document]
Comment (jasonnicholson) [reply mingw-w64-git-lfs/PKGBUILD:55]: Sure, give me a minute.

**Review by dscho — APPROVED** [review]

**Comment by dscho** [comment]
/add release note feature The Git for Windows installer now [also includes](https://github.com/git-for-windows/build-extra/pull/491) the Git LFS documentation (i.e. `git help git-lfs` now works).

**Pull request merged**